globals/strings.gd

```diff
@@ -675,6 +692,8 @@ func get_condition_text(conditions):
 				if len(required_bloom_levels) > 0:
 					bloom_str = " " + tr("(Bloom %s)") % "/".join(required_bloom_levels)
 				text += "If attached to %s%s: " % [get_names([condition["required_member_name"]])[0], bloom_str]
+			"attached_with":
```

Comment:
if you change this to "has_attached_card", change this too and move it in alpha order preferably